labgraph/examples/labgraph_monitor_example.py

```diff
@@ -0,0 +1,131 @@
+import labgraph as lg
```

Comment:
Since Labgraph Monitor currently isn't part of LabGraph core, I suggest moving this example to **extensions/yaml_support/labgraph_monitor/examples**

Reply:
That would makes sense. Done